apps/lib/apps.c

```diff
@@ -2518,46 +2528,21 @@ BIO *bio_open_owner(const char *filename, int format, int private)
     return NULL;
```

Comment:
Could it be that bio_open_owner should also follow the new model?
in one code path it returns bio_open_default and the other path
requires the caller to check for errors?
What do you think?

Reply:
I took a quick look and it didn't seem easy, so I'm not opposed but I'd rather do it in a separate PR.  Can't do the fopen/new_fp trick because of what it does, but yeah, making it return or fail/exit makes sense.

Reply:
More important is probably 1.1.1 back-port, most of the time that is forgotten nowadays.

Reply:
Actually, restructuring the code and it was easy.  Will be in an additional commit shortly.